### PR TITLE
untransianted required properties

### PR DESCRIPTION
**Pull request by michaelconnor468**

### CODE REVIEW

File: src/foam/nanos/session/Session.js [hunk 1]
```diff
@@ -51,7 +51,8 @@ foam.CLASS({
     },
     {
       class: 'Long',
-      name: 'uses'
+      name: 'uses',
+      transient: true
```
Comment (TW80000) [inline src/foam/nanos/session/Session.js]: Sorry, this should just be `storageTransient`, not `networkTransient` as well. If it were `networkTransient` then admins wouldn't be able to see how many uses sessions have.

**Review by TW80000 — CHANGES_REQUESTED** [review]

**Review by TW80000 — APPROVED** [review]

**Review by jlhughes — COMMENTED** [review]
I would expect lastUsed to also required storageTransient.  
See discussion in https://github.com/foam-framework/foam2/pull/2542

**Review by jlhughes — CHANGES_REQUESTED** [review]
Make lastUsed storageTransient. 
On restart the session will effectively be reset.  We'll err with this rather than write every session update for now.

**Review by jlhughes — APPROVED** [review]

File: src/foam/nanos/session/Session.js [hunk 2]
```diff
@@ -65,6 +66,7 @@ foam.CLASS({
     {
       class: 'Long',
       name: 'uses'
```
Comment (jlhughes) [inline src/foam/nanos/session/Session.js]:
```suggestion
      name: 'uses',
```

**Pull request merged**